storybook/.storybook/analysis.js

```diff
@@ -1,6 +1,6 @@
 import { knob } from '@storybook/addon-knobs';
 
-import ui from '@nuxeo/nuxeo-ui-elements/analysis.json';
+import ui from '../../ui/analysis.json';
```

Comment:
Why changing to relative path?

Reply:
actually we don't need to do it, I'm going to revert it.